spec/schema.json

```diff
@@ -55,6 +55,48 @@
         }
       }
     },
+    "/transactions/{transaction_id}/overseas-entity/{overseas_entity_id}/validation-status": {
+      "parameters": [
+        {
+          "name": "transaction_id",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        },
+        {
+          "name": "overseas_entity_id",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }
+      ],
+      "get": {
+        "summary": "Verify that all the tasks for the Specified Overseas Entities Submission have been completed",
```

Comment:
in the summary, 'all the tasks' relates to the confirmation-statement tasks that the user confirms.For Overseas Entities, just have 'Verify that the specified Overseas Entities submission is valid'.
Same for the 200 description below - "The specified Overseas Entities Submission is valid."